client/src/components/APIDataTypes.ts

```diff
@@ -19,6 +19,17 @@ type HpdOwnerContact = {
   value: string;
```

Comment:
I know this isn't part of the original PR, but calling this `value` is weird to me, I feel like it should be `fullName` or something.  But if we've already got production code looking at this field then definitely don't worry about it!!

Reply:
@toolness I agree! it's a horribly vague name for this property that unfortunately predates my time working on WOW... maybe we can file an issue to try and address this in the future?

Reply:
Oh gotcha yeah no worries!!